node/package.json

```
@@ -4,10 +4,12 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "apollo-server": "^2.8.1",
+    "apollo-server-errors": "^2.3.1",
```

Remove this dependency with `yarn remove apollo-server-errors`
```suggestion
    "apollo-server-errors": "^2.3.1",
```